Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/Taa.azsl

```diff
@@ -99,11 +99,20 @@ float3 YCoCgToRgb(float3 yCoCg)
 // This function samples a 4x4 neighborhood around the uv. By taking advantage of bilinear filtering this can be
 // done with only 9 taps on the edges between pixels. The cost is further reduced by dropping the 4 diagonal
 // samples as their influence is negligible.
-float4 SampleCatmullRom5Tap(Texture2D<float4> texture, SamplerState linearSampler, float2 uv, float2 textureSize, float2 rcpTextureSize, float sharpness)
+float3 SampleCatmullRom5Tap(Texture2D<float4> texture, SamplerState linearSampler, float2 uv, float2 textureSize, float2 rcpTextureSize, float sharpness)
 {
     // Think of sample locations in the 4x4 neighborhood as having a top left coordinate of 0,0 and
     // a bottom right coordinate of 3,3.
 
+    // Use the local area green value as a measurement of luminance, and tweak the sharpness
+    // based on the difference between the max and min values. This will reduce the contribution
+    // of negative weights in high contrast areas and help prevent ringing artifacts.
```

Comment:
super nit: start with two lines stating of the problem it comes to solve.

Reply:
Do you mean what this function is trying to solve? (there's a description above the function definition) or what this particular piece of code is solving (meaning you suggest re-wording this comment to state the problem first)?

Reply:
Updated the comment to be more clear.